pdg/src/snapshots/c2rust_pdg__tests__analysis_test_pdg_snapshot.snap

```diff
@@ -87,51 +87,50 @@ g {
-	n[28]: copy       n[27] => _7  @ bb1[4]:  fn recur;  _7 = move _8 as *mut libc::c_void (Misc);
-	n[29]: free       n[28] => _0  @ bb1[5]:  fn recur;  _0 = free(move _7);                      
-	n[30]: copy       n[26] => _14 @ bb9[5]:  fn recur;  _14 = _2;                                
-	n[31]: copy       n[26] => _14 @ bb9[5]:  fn recur;  _14 = _2;                                
-	n[32]: copy       n[26] => _14 @ bb9[5]:  fn recur;  _14 = _2;                                
-	n[33]: addr.load  n[1]  => _   @ bb5[3]:  fn simple; _17 = (*_5);                             
-	n[34]: addr.store n[3]  => _   @ bb5[7]:  fn simple; (*_1) = move _18;                        
-}
-nodes_that_need_write = [34, 18, 17, 11, 10, 9, 8, 5, 4, 3, 2, 1, 0]
+	n[0]: alloc   _    => _2 @ bb1[2]: fn simple; _2 = malloc(move _3);                   
+	n[1]: copy    n[0] => _1 @ bb2[2]: fn simple; _1 = move _2 as *mut pointers::S (Misc);
+	n[2]: field.0 n[1] => _9 @ bb4[5]: fn simple; _9 = &raw const ((*_1).0: i32);         
+}
+nodes_that_need_write = []
+
+g {
+	n[0]:  alloc       _     => _6     @ bb3[2]:  fn simple; _6 = malloc(move _7);                    
+	n[1]:  copy        n[0]  => _5     @ bb4[2]:  fn simple; _5 = move _6 as *mut pointers::S (Misc); 
+	n[2]:  copy        n[1]  => _10    @ bb4[9]:  fn simple; _10 = _5;                                
+	n[3]:  copy        n[2]  => _1     @ bb4[10]: fn simple; _1 = move _10;                           
+	n[4]:  field.0     n[3]  => _      @ bb4[11]: fn simple; ((*_1).0: i32) = const 10_i32;           
+	n[5]:  addr.store  n[4]  => _      @ bb4[11]: fn simple; ((*_1).0: i32) = const 10_i32;           
+	n[6]:  field.0     n[3]  => _11    @ bb4[13]: fn simple; _11 = ((*_1).0: i32);                    
+	n[7]:  addr.load   n[6]  => _      @ bb4[13]: fn simple; _11 = ((*_1).0: i32);                    
+	n[8]:  field.0     n[1]  => _      @ bb4[14]: fn simple; ((*_5).0: i32) = move _11;               
+	n[9]:  addr.store  n[8]  => _      @ bb4[14]: fn simple; ((*_5).0: i32) = move _11;               
+	n[10]: field.1     n[3]  => _      @ bb4[16]: fn simple; ((*_1).1: u64) = const 9_u64;            
+	n[11]: addr.store  n[10] => _      @ bb4[16]: fn simple; ((*_1).1: u64) = const 9_u64;            
+	n[12]: field.0     n[3]  => _12    @ bb4[18]: fn simple; _12 = ((*_1).0: i32);                    
+	n[13]: addr.load   n[12] => _      @ bb4[18]: fn simple; _12 = ((*_1).0: i32);                    
+	n[14]: field.1     n[3]  => _13    @ bb4[21]: fn simple; _13 = &raw const ((*_1).1: u64);         
+	n[15]: copy        n[3]  => _14    @ bb4[25]: fn simple; _14 = &raw const (*_1);                  
+	n[16]: field.2     n[3]  => _      @ bb4[25]: fn simple; ((*_1).2: *const pointers::S) = move _14;
+	n[17]: addr.store  n[16] => _      @ bb4[25]: fn simple; ((*_1).2: *const pointers::S) = move _14;
+	n[18]: value.store n[15] => _1.*.2 @ bb4[26]: fn simple; ((*_1).2: *const pointers::S) = move _14;
```

Comment:
Something must have gone wrong here - it's showing two different statements at the same position:
```
bb4[25]: fn simple; _14 = &raw const (*_1);                  
bb4[25]: fn simple; ((*_1).2: *const pointers::S) = move _14;
```
Followed by the same statement at two different positions:
```
bb4[25]: fn simple; ((*_1).2: *const pointers::S) = move _14;
bb4[26]: fn simple; ((*_1).2: *const pointers::S) = move _14;
```

Reply:
Made an issue here https://github.com/immunant/c2rust/issues/581